unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs

```diff
@@ -127,6 +120,9 @@ void OnDestroy()
         private void Update()
         {
             InputController_Legacy.i.Update();
+            if (interactionHoverCanvasController != null) {
+                Environment.i.pointerEventsController.Update(interactionHoverCanvasController);
+            }
```

Comment:
We should remove the braces here or move braces to the next line to abide with the style of the rest of the project.

Reply:
I love consistency... but hate single-line-braceless ifs

Reply:
Done